docker/db/init.sh

```
@@ -0,0 +1,38 @@
+#!/bin/bash
+set -e
+
+MYSQL_CMD="mysql -u root -p$MYSQL_ROOT_PASSWORD"
+
+echo "Initialisation de la base de données..."
+
+echo "Sélection de la base de données ${MYSQL_DATABASE}..."
+$MYSQL_CMD -e "USE ${MYSQL_DATABASE};" || {echo "Erreur: la base de données ${MYSQL_DATABASE} n'existe pas"; exit 1;}
```

Shellcheck (0.10.0)

[error] You need a space after the '{'.

(SC1054)